opengl.gpr

```diff
@@ -70,7 +70,7 @@ library project OpenGL is
    end Builder;
 
    package Compiler is
-      Base_Switches := ("-gnat05", "-gnatwa", "-gnatwl", "-gnatwe",
+      Base_Switches := ("-gnat12", "-gnatwa", "-gnatwl", "-gnatwe",
```

Comment:
I assume this is because GNAT Community now only supports the latest Ada standard. However, I am not aware that anyone uses this with a different compiler, so I guess we could as well go with it.